packages/@react-aria/color/src/useColorArea.ts

```diff
@@ -295,24 +312,38 @@ export function useColorArea(props: ColorAreaAriaProps, state: ColorAreaState):
           onThumbDown(e.changedTouches[0].identifier);
         }
       })
-  }, keyboardProps, movePropsThumb);
+  }, focusWithinProps, keyboardProps, movePropsThumb);
 
   let isMobile = isIOS() || isAndroid();
 
+  function getAriaLabelForChannel(channel: ColorChannel): string {
+    return isMobile ? state.value.getChannelName(channel, locale) : formatMessage('colorPicker');
+  }
+
+  let colorPickerLabel = formatMessage('colorPicker');
+  let xChannelLabel = getAriaLabelForChannel(xChannel);
+  let yChannelLabel = getAriaLabelForChannel(yChannel);
+
   let xInputLabellingProps = useLabels({
     ...props,
-    'aria-label': isMobile  ? state.value.getChannelName(xChannel, locale) : formatMessage('x/y', {x: state.value.getChannelName(xChannel, locale), y: state.value.getChannelName(yChannel, locale)})
+    'aria-label': ariaLabel ? `${ariaLabel} ${xChannelLabel}` : xChannelLabel
```

Comment:
is this order true for all locales?

Reply:
This is the equivalent to the ordering of IDREFs using `aria-labelledby`. `ariaLabel` is the external label, like "foo" in the Storybook example that demonstrates labelling using `aria-label`. the `xChannelLabel` is either "Color picker", on desktop or the channel name, e.g. "Green", on mobile.

Reply:
right, I get that it's the same ordering you'd get joining id's together for aria-labelledby, but doesn't the screen reader put commas between each id in that case? or some sort of pause? here, we're just putting a space?

Reply:
I could add a comma, if you think it would be appropriate.